.github/workflows/sycl-linux-run-tests.yml

```diff
@@ -78,6 +101,14 @@ on:
         default: 'false'
         required: False
 
+    outputs:
+      run_conclusion:
```

Comment:
could be im just dense but its not clear to me what possible values of this be from the var name. maybe we can rename it?